normandy/control/views.py

```diff
@@ -0,0 +1,18 @@
+from django.views import generic
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from normandy.recipes.models import Recipe
+
+
+class IndexView(LoginRequiredMixin, generic.ListView):
```

Comment:
My general impression of class-based-views (CBV)  and Django generics are that CBVs are a great idea, and that Django's CBVs often don't do things the way I would like. However, it seems like what you have here exactly matches what the generics are doing, so might as well use them. We can always move away from them in the future, if needed.